docs/modules/ROOT/pages/usage.adoc

```diff
@@ -194,6 +194,51 @@ In the above example, all data nodes in the default group will store data (the l
 
 By default, in case nothing is configured in the custom resource for a certain role group, each Pod will have a `1Gi` large local volume mount for the data location.
```

Comment:
I believe the default changed
```suggestion
By default, in case nothing is configured in the custom resource for a certain role group, each Pod will have a `5Gi` large volume mount for the data location.
```

Reply:
thx!